releng/platform.target

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?><target name="ELS Target" sequenceNumber="14">

```

Comment:
I'd rather the file to be called `ide.target` as it will make clearer that this .target isn't intended to be used by Tycho to resolve dependencies.